pkg/client/client.go

```diff
@@ -185,7 +186,7 @@ func (c *Client) SubjectAccessReviews(namespace string) SubjectAccessReviewInter
 	return newSubjectAccessReviews(c, namespace)
 }
 
-// RootSubjectAccessReviews provides a REST client for RootSubjectAccessReviews
+// RootSubjectAccessReviews provides a REST client for SubjectAccessReviews
 func (c *Client) RootSubjectAccessReviews() SubjectAccessReviewInterface {
```

Comment:
> We need to settle on terminology.

@smarterclayton Opinion?  I have no objection to `Cluster` (it's what I used in https://github.com/openshift/origin/pull/1989), but I'd rather not go through the renaming more than necessary....